app/src/main/java/cs465/illinois/edu/dogonthequad/CreateMeetupActivity.java

```diff
@@ -52,11 +53,16 @@ protected void initialize(int resourceId) {
         Log.d(MEETUP_KEY, "In CreateMeetup, received meetup: " + json);
         mMeetup = new Gson().fromJson(json, Meetup.class);
 
+        try {
+            mBackButton = findViewById(R.id.back_button);
+        } catch (RuntimeException e) {
+            mBackButton = findViewById(R.id.cancel_button);
```

Comment:
This line is returning `null`, even though the layout contains `cancel_button`.

Reply:
I'm confused why you're trying to manage the cancel button within CreateMeetupActivity rather than CreateMeetupReview. This functionality should definitely be separate, because it needs to create a popup and then go back to the map screen. 

The problem is that FindViewById doesn't throw an exception when it fails to find the view, it just returns null. 
[Link](https://developer.android.com/reference/android/view/View.html#findViewById(int))

The try/catch works for next/confirm because after finding next we try to do stuff to it, and then catch the failure of trying to set text or whatever.

Random side note: the mIsNextButtonEnabled thing doesn't actually enable/disable the next button and shouldn't really be in CreateMeetupActivity. Check out how it works in SelectDog to actually gray out the button.

Reply:
Why should there be default behavior for the cancel button? I could be mistaken, but I don't think defining another onClick overrides--it just registers an additional listener, and both listeners would be called (I think?) Either way, we don't need the default behavior, so it shouldn't be there.

Reply:
I agree, I guess it doesn't have much purpose. It does the way I've implemented it, but the default is removed now. (Only the `CreateMeetupReviewActivity.onClick` is called because it overrides `CreateMeetupActivity.onClick`, and so it's only available using `super.onClick`.)